system/jlib/jsocket.hpp

```diff
@@ -30,7 +30,7 @@
 #include "jexcept.hpp"
 #include "jthread.hpp"
 
-#if 1 // spurious mplink closed tracing info
+#if 0 // spurious mplink closed tracing info
```

Comment:
trivial: perhaps should comment out as in other change (in mpcomm.cpp) to be consistent.